### ISSUE TITLE
support checking the database files

### ISSUE THREAD

**Issue by nicolas33** [OfflineIMAP/offlineimap#402]
Something like `--check-databases` would check that:

* all the entries can be read
* one fake entry can be inserted
* the fake entry can be removed

**Comment by kanishchugh** [comment]
Hi, @nicolas33 , 
I would like to work on adding this feature. also can you provide me with more details about it?
Thanks.

**Comment by yash07bagul** [comment]
hey! I am a beginner to open source and wants to contribute to open source , This problem looks best fit for me . Is it open to contribute , if so please inform me.

**Comment by chris001** [comment]
Yes, you're welcome to contribute with a Pull Request, however, take note to please make sure your code works on both `offlineimap` (python 2 version in this repo) and `offlineimap3` (python 3 version, found in https://github.com/OfflineIMAP/offlineimap3/ ).

**Comment by yash07bagul** [comment]
Hey! Thank you so much for a reply and so sorry for a asking so silly question and wasting your time but I am not able to understand exactly what I have to do particularly for this issue ?

**Comment by chris001** [comment]
Add the `--check-databases` command to the command-line that will do the 3 checks that it says in the first message, on the [local status cache sqlite database](https://github.com/OfflineIMAP/offlineimap/blob/master/docs/offlineimap.txt#L224).

**Comment by JennyPoudel** [comment]
I am trying to get started on opensource, can u please assign me with an issue of it is still open ,thanks a lot!
